static/app/utils/discover/fieldRenderers.tsx

```diff
@@ -338,7 +348,30 @@ const SPECIAL_FIELDS: SpecialFields = {
             }}
             items={items}
           />
-        </Container>
+        </RightAlignedContainer>
+      );
+    },
+  },
+  minidump: {
+    sortField: null,
+    renderFunc: data => {
+      const attachments: Array<IssueAttachment & {url: string}> = data.attachments;
+
+      const minidump = attachments.find(
+        attachment => attachment.type === 'event.minidump'
+      );
+
+      return (
+        <RightAlignedContainer>
+          <Button size="xs" disabled={!minidump ? true : false}>
```

Comment:
nit: `!minidump` is already a boolean so you don't have to do a ternary here, just `disables={!minidump}` should suffice

Reply:
Ohh right, I was using the string true and false for something else before. Anyways, will fix